docs/tutorials/custom/custom-base-dog.md

```diff
@@ -167,7 +164,7 @@ Save the file.
 
 Go to the [robot page](https://app.viam.com/robots) for your robot dog that you created when installing `viam-server` on the Pi.
 
-In the banner towards the top of the page, the IP address of the robot dog Pi is displayed under **ips**.
+In the banner towards the top of the page, the IP address of the robot dog Pi is displayed under **IPs**.

```

Comment:
could you just use 127.0.0.1 (localhost) since this test script is running on the same machine as the freenove server?

Reply:
Not here, because dog_test.py is being run on a different  computer

Reply:
I could rewrite this to have them run the script on the local machine instead of the laptop but I don't think it matters so I'd rather leave it as-is.
Maybe it could be changed in the configured component (like maybe I don't even need that to be a configurable attribute, but could just hardcode it as localhost), but as-written, and as-tested, this way works. Also, then maybe it gives them some flexibility with where they're running things??

Reply:
so if you don't want it to run on the same machine, can't you use robotdog.local instead of the IP address?